pyquil/operator_estimation.py

```diff
@@ -751,8 +751,7 @@ def serializable(self):
 def measure_observables(qc: QuantumComputer, tomo_experiment: TomographyExperiment,
                         n_shots: int = 1000, progress_callback=None, active_reset=False,
                         readout_symmetrize: str = None, calibrate_readout: str = None,
-                        inherit_povm: bool = True, inherit_gate_defn: bool = False,
-                        inherit_defined_gate_app: bool = False, inherit_kraus: bool = False):
+                        inherit_readout_error: bool = True, inherit_gate_noise: bool = True):
```

Comment:
I thought we agree this functionality should not be exposes to the user -- readout error and gate noise should always be inherited, as long as error models can be specified in programs.

Reply:
> No, I am looking at the latest version of the PR -- the arguments `inherit_readout_error: bool = True, inherit_gate_noise: bool = True` are in the function signature, and should not be. Please remove them as discussed.
> 
> I also though you intended to make the default behavior to have symmetrization and calibration turned on, but instead you have `readout_symmetrize: str = None, calibrate_readout: str = None`.

Ah yes, working on those. I'm currently writing some more tests. Will ping you when the changes we discussed are in.